packages/@react-aria/button/src/useButton.ts

```diff
@@ -42,18 +42,21 @@ export function useButton(props: AriaButtonProps, ref: RefObject<HTMLElement>):
     onClick: deprecatedOnClick,
     href,
     target,
-    type = 'button'
+    type = 'button',
+    rel = []
   } = props;
   let additionalProps;
   if (elementType !== 'button') {
+    let relString = Array.isArray(rel) ? rel.join(' ') : rel;
     additionalProps = {
       role: 'button',
       tabIndex: isDisabled ? undefined : 0,
       href: elementType === 'a' && isDisabled ? undefined : href,
       target: elementType === 'a' ? target : undefined,
       type: elementType === 'input' ? type : undefined,
       disabled: elementType === 'input' ? isDisabled : undefined,
-      'aria-disabled': !isDisabled || elementType === 'input' ? undefined : isDisabled
+      'aria-disabled': !isDisabled || elementType === 'input' ? undefined : isDisabled,
+      rel: elementType === 'a' ? relString : ''
```

Comment:
can we return undefined instead of an empty string?
this should be simpler if rel doesn't have a default value

`let relString = Array.isArray(rel) ? rel.join(' ') : rel;`
this would then be a joined string, a string, or undefined
then when we put it into additionalProps
`rel: elementType === 'a' ? relString : undefined`, now it won't appear as an attribute at all for React unless it has a value